frontend/src/utils.ts

```diff
@@ -2,3 +2,14 @@ export type someKeysOf<T> = { [key in keyof T]?: T[key] };
 export type WorkspaceRole = 'user' | 'manager';
 export type BadgeSize = 'small' | 'middle' | 'large';
 export type BoxHeaderSize = 'small' | 'middle' | 'large';
+export type Template = {
+  id: string;
+  name: string;
+  gui: boolean;
+  instances: Array<{
+    id: number;
+    name: string;
+    ip: string;
+    status: boolean;
+  }>;
```

Comment:
Was wondering if making an _Instance_ interface could make things more organized...

Reply:
it could be a good idea, i do the changes you suggest